docs/_includes/attr-data.adoc

```diff
@@ -2,58 +2,141 @@
 Included in:
 
 - user-manual: Built-in data attributes
+
+Changes:
+Added an example/notes column; 
+deleted doctitle because it is documented elsewhere; 
+added basebackend, filetype and outdir;
+consistently use 'document' for the input file and 'output file' for the output.
+added safe-mode stuff
```

Comment:
I don't think we need these changes in the document itself. It's sufficient to put this information in the comments of the pull request.